spec/models/base_item_spec.rb

```diff
@@ -15,37 +15,23 @@
 
 require "rails_helper"
 
-RSpec.describe BaseItem, type: :model do
-  describe "Validations >" do
-    it "is invalid without a name" do
-      expect(build(:base_item, name: nil)).not_to be_valid
-    end
-
-    it "is invalid without a unique name" do
-      f = create(:base_item)
-      expect(build(:base_item, name: f.name)).not_to be_valid
-    end
-
-    it "is invalid without a partner key" do
-      expect(build(:base_item, partner_key: nil)).not_to be_valid
-    end
+RSpec.describe BaseItem, type: :model, skip_seed: true do
+  let(:organization) { create(:organization, skip_items: true) }
 
-    it "is invalid without a uniqueness key" do
-      f = create(:base_item)
-      expect(build(:base_item, partner_key: f.partner_key)).not_to be_valid
-    end
+  describe "Validations >" do
+    it { should validate_presence_of(:name) }
+    it { should validate_uniqueness_of(:name) }
+    it { should validate_presence_of(:partner_key) }
+    it { should validate_uniqueness_of(:partner_key) }
   end
 
   describe "Associations >" do
     it "keeps count of its associated items" do
-      c = BaseItem.first
+      c = create(:base_item, name: "Base", item_count: 0)
```

Comment:
Should `item_count` default to 0 instead of `nil`?

Reply:
Not sure it makes much difference tbh, except in this specific spec :grin: